flight/Libraries/math/misc_math.c

```diff
@@ -367,6 +367,27 @@ uint32_t randomize_int(uint32_t interval)
 	return rand_val % interval;
 }
 
+/**
```

Comment:
This is factored to misc_math, because it's now used by the actuator subsystem. (And because it's a generally applicable bit of code).